docs/content/api_core/api_Maps.rst

```diff
@@ -61,10 +61,10 @@ done by the :class:`SimPEG.Maps.ExpMap` described above.
     m = np.r_[0.2,1,0.1,2,2.9] # only 5 model parameters!
     sig = myMap * m
 
-.. plot::
 
-    from SimPEG import Examples
-    Examples.Maps_ComboMaps.run()
+.. image:: /content/examples/maps/images/sphx_glr_plot_combo_001.png
```

Comment:
This is the way that I have gone around running the example twice in the doc build. Just have an image and link through to the example. The test should catch any broken links.